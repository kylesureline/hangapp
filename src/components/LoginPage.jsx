import React from 'react';
import Login from './Login.jsx';

const LoginPage = () => (
  <div className="box-layout">
    <div className="box-layout__box">
      <h1 className="box-layout__title">Login</h1>
      <Login />
    </div>
  </div>
);

export default LoginPage;
