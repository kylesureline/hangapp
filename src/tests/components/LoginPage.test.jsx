import React from 'react';
import { shallow } from 'enzyme';
import LoginPage from '../../components/LoginPage.jsx';

test('should render LoginPage correctly', () => {
  const wrapper = shallow(<LoginPage />);
  expect(wrapper).toMatchSnapshot();
});
