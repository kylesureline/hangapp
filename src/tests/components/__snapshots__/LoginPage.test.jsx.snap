// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoginPage correctly 1`] = `
<div
  className="box-layout"
>
  <div
    className="box-layout__box"
  >
    <h1
      className="box-layout__title"
    >
      Login
    </h1>
    <Connect(Login) />
  </div>
</div>
`;
